### PR TITLE
Clarify entitlement to free corona test (Addresses #2607)

### LINKED ISSUE #2607: Out of date information in `vac_cert_red_warning` & `voluntary_self_quarantine` reg. entitlement to free corona test with increased risk encounter(s)
## Where to find the issue
- https://www.coronawarn.app/en/faq/results/#vac_cert_red_warning
- https://www.coronawarn.app/de/faq/results/#vac_cert_red_warning
- https://www.coronawarn.app/en/faq/results/#voluntary_self_quarantine 
- https://www.coronawarn.app/de/faq/results/#voluntary_self_quarantine

## Describe the issue
The FAQ entry states: 
"In case of an increased risk warning (red tile), you are entitled to a free test. This does not necessarily have to be a PCR test."

This statement is no longer true, see https://github.com/corona-warn-app/cwa-documentation/issues/877.

## Suggested change
Remove the sentence.

## Additional information
PR follows soon.

### PR DESCRIPTION
**Pull request by Ein-Tim**
## Description

This PR clarifies the entitlement to a free corona test in the FAQ entries `vac_cert_red_warning` & `voluntary_self_quarantine`.

## What was changed

- https://www.coronawarn.app/en/faq/results/#vac_cert_red_warning
- https://www.coronawarn.app/de/faq/results/#vac_cert_red_warning
- https://www.coronawarn.app/en/faq/results/#voluntary_self_quarantine 
- https://www.coronawarn.app/de/faq/results/#voluntary_self_quarantine

## Screenshots

`vac_cert_red_warning`

| German | English |
|---|---|
|vac_cert_red_warning GERMAN | vac_cert_red_warningn ENGLISH |

---

`voluntary_self_quarantine`

| German | English |
|---|---|
|voluntary_self_quarantine GERMAN | voluntary_self_quarantine ENGLISH |

---

**Fixes #2607**

### CODE REVIEW

**Comment by Ein-Tim** [comment]
I will attempt running the cypress test suite on this PR now. Let's see if I can do that successfully.

**Comment by Ein-Tim** [comment]
So, one hour later, Cypress tests have passed. There were 5 issue, three false positives and two reg. http://localhost:8000/assets/documents/setupEventQRCodeGenerator.bat.

```
assert Link: http://localhost:8000/assets/documents/setupEventQRCodeGenerator.bat: expected false to equal true
```

No idea if this is related to this PR or if this is a general problem. IMO this PR is good to be merged.

File: src/data/faq_de.json [hunk 1]
```diff
@@ -1138,7 +1138,7 @@
...
+                                    "Sollte die Corona-Warn-App Ihnen eine Begegnung mit erhöhtem Risiko in einer roten Kachel anzeigen, begeben Sie sich, wenn möglich, in eine freiwillige Selbstquarantäne, bis Sie ein negatives Testergebnis vorweisen können. Dies gilt auch, wenn Sie geimpft oder genesen sind und keine Symptome zeigen. Nach <a href='https://www.gesetze-im-internet.de/coronatestv_2021-10/__4a.html' target='_blank' rel='noopener'>§4a Testverordnung</a> haben Sie Anspruch auf einen kostenlosen Schnelltest. Weitere Infomrationen dazu finden Sie hier: <a href='https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html#c23926' target='_blank' rel='noopener'>Fragen und Antworten zu COVID-19 Tests (BMG)</a>.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:1141]: "Infomrationen" is a typo

File: src/data/faq_de.json [hunk 2]
```diff
@@ -2245,7 +2245,7 @@
...
...egie/faq-covid-19-tests.html#c23926' target='_blank' rel='noopener'>Fragen und Antworten zu COVID-19 Tests (BMG)</a>. Das muss nicht notwendigerweise ein PCR-Test sein.</li><li>Empfohlen sind mindestens 7 Tage, besser 10 Tage, für die Zeitdauer der Selbstquarantäne ab dem Tag der letzten Risiko-Begegnung. Das Datum der letzten Begegnung mit erhöhtem Risiko finden Sie in der roten Kachel.</li><li>Verzichten Sie nach Möglichkeit auf Besuche und Aktivitäten, bei denen Sie mit anderen Menschen zusammenkommen. Lassen Sie sich Lebensmittel liefern oder bitten Sie jemanden, die Einkäufe zu erledigen; meiden Sie öffentliche Verkehrsmittel und verzichten Sie auf Flug- und Zugreisen.</li><li>Vermeiden Sie insbesondere Kontakte zu vulnerablen Personen und Gruppen, beispielsweise zu Älteren und Vorerkrankten.</li><li>Minimieren Sie auch Kontakte im eigenen Haushalt.</li><li>Wenn Ihr Arbeitgeber Ihnen die Möglichkeit gibt, arbeiten Sie von zu Hause aus.</li><li>Achten Sie mindestens für 10 Tage (ab Risiko-Begegnung) auf Krankheitssymptome und wenden Sie sich an Ihre Hausarztpraxis, falls Sie sich krank fühlen oder Symptome auftreten sollten.</li><li>Halten Sie die Hygieneregeln ein.</li></ul>",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:2248]: "Infomrationen" is a typo

Comment (Ein-Tim) [reply src/data/faq_de.json]: Typo has been fixed in [`070ebd2`](https://github.com/corona-warn-app/cwa-website/pull/2921/commits/070ebd26220fa47e63bc42bee9cd03452232076d). Thanks for spotting! Please re-review.

File: src/data/faq_de.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq_de.json]: Typo has been fixed in [`070ebd2`](https://github.com/corona-warn-app/cwa-website/pull/2921/commits/070ebd26220fa47e63bc42bee9cd03452232076d). Thanks for spotting! Please re-review.

**Review by GisoSchroederSAP — APPROVED** [review]
Even is the existing article war absolutely correct and clear, according to §1 and §4a of TestV, I accept the changes. Please note, however, that §4a is addressing asymptomatic persons only, instead §1 covers a persons with health insurance and already states "Zur Diagnostik können nach Maßgabe dieser Verordnung und der Teststrategie des Bundesministeriums für Gesundheit eine Labordiagnostik mittels Nukleinsäurenachweis, eine 
variantenspezifische PCR-Testung **oder** eine Diagnostik durch Antigen-Test gehören. Zur 
Diagnostik durch Antigen-Test gehören 
1. eine Labordiagnostik mittels Antigen-Test, 
2. ein Antigen-Test zur patientennahen Anwendung durch Dritte (PoC-Antigen-Test) oder 
3. ein Antigen-Test zur Eigenanwendung, dessen Durchführung von einem Leistungserbringer nach § 6 vor Ort überwacht wird (überwachter Antigen-Test zur Eigenanwendung). "
So there is neither a real limitation nor vagueness. At no point a PCR test is promised.
But if you think, the new wording helps to guide the users to the one-and-only-option Antigene-Bürgertest, please go ahead.

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 @Ein-Tim Thanks!

**Pull request merged**